test/integration/visapi.ts

```diff
@@ -0,0 +1,39 @@
+import { db } from "../../src/conn";
+import * as visapi from "../../src/visapi";
+import * as chai from "chai";
+import * as fs from "fs";
+import * as path from "path";
+
+export default function() : void {
+  describe("Visapi Tests", function(){
+    before("Add testing data to database", function() {
+      return new Promise((resolve, reject)=>{
+        fs.readFile(path.join(__dirname, "10087-Saloon-8aca7.json"),(err, data)=>{
+            db("game").insert({
+            status: 'finished',
+            gamelog: data,
+            visualized: false
+          }).then(resolve)
+            .catch(reject);
+        })
```

Comment:
These lines are actually terminating lines

`fs.readFile(...).then(...);`
`fs.readFile(...).then(...).then(...);`
`fs.readFile(...).then(...).then(...).catch(...);`

as a result, they should have semicolons at the end

typically any `})` set will need a semicolon